lib/withOnyx.js

```diff
@@ -59,7 +59,7 @@ export default function (mapOnyxToState, shouldDelayUpdates = false) {
                     (resultObj, mapping, propertyName) => {
                         const key = Str.result(mapping.key, props);
                         let value = Onyx.tryGetCachedValue(key, mapping);
-                        if (!value && mapping.initialValue) {
+                        if (value === undefined) {
```

Comment:
This change caused a couple of regressions (e.g. https://github.com/Expensify/App/issues/37743, https://github.com/Expensify/App/issues/37746) and was eventually reverted.